make/autoconf/flags-ldflags.m4

```diff
@@ -27,7 +27,7 @@
 #
 
 # ===========================================================================
-# (c) Copyright IBM Corp. 2019, 2019 All Rights Reserved
+# (c) Copyright IBM Corp. 2019, 2024 All Rights Reserved
```

Comment:
Please omit this change.

Reply:
Removed this copyright year change.
Also restored `make/Main.gmk` which removed `test-image-lib` target.